src/Engine/EngineBase.php

```diff
@@ -201,4 +214,41 @@ protected function getInvalidLangsWarning(array $invalidLangs): string
             [ 'variables' => [ $this->intuition->listToText($invalidLangs) ] ]
         );
     }
+
+    /**
+     * @param string $imageUrl The original image URL.
+     * @param int[] $crop Array with keys `x, `y`, `width` and `height`.
+     * @param ?bool $fetch Whether to download the image or not.
+     * @return Image
+     * @throws OcrException If the image couldn't be fetched.
+     */
+    public function getImage(string $imageUrl, array $crop, ?bool $fetch = false): Image
```

Comment:
Suggest using class constants for $fetch, e.g. FETCH_DOWNLOAD or something (naming things is hard...)